cmd/singularity/testdata/deffile.tmpl

```diff
@@ -85,4 +85,62 @@ Namespace: {{.Namespace}}
 {{- end}}
 {{- range $l := .StartScript}}
     {{$l}}
+{{- end}}
+
+
+
```

Comment:
Are all this white lines required?

Reply:
No it was just for visual separation between regular sections and apps. The file generated from this template won't have these spaces